wordpress-mobile/WordPress-Android
# [Jetpack Content Migration Flow] Add static UI to Welcome Screen (2)

Fixes Partially #17368
Depends on: #17387, which depends on #17386
`Ready for merge` after #17387

Adds UI elements except the sites list to the Welcome Screen after WordPress to Jetpack auto login, which is behind a temporary feature flag.

To test:

0. **Prerequisite:** Toggle the following feature flags `on` in `Wordpress/build.gradle` code:
   (it's not possible to do this from the app because we need to be logged in to switch feature flags):
   - Set `JETPACK_SHARED_LOGIN` to `true`
   - Set `JETPACK_PROVIDER_SYNC` to `true`
   - Set `JETPACK_MIGRATION_FLOW` to `true`
   - Press the `Sync Now` button that appears at the top right of the file editing view in Android Studio
1. Build & Install the WordPress app then Login
   _Note: If testing this right after [dependency PR (1)](https://github.com/wordpress-mobile/WordPress-Android/pull/17387) it's enough to uninstall the Jetpack app and skip step 1._
2. Build & Install the Jetpack app **using the same variant**
   (e.g. if WordPress was `wordpressWasabiDebug` , use `jetpackWasabiDebug`)
3. **Expect** to see the "Welcome to Jetpack!" screen without the site list
   - Buttons have no functionality on click

### Screenshot
| <img width="314" src="https://user-images.githubusercontent.com/4588074/198841635-0240ad3e-a4dd-46c7-aa54-4be780848c10.png"> |
| - |

### Merge Strategy
1. Merge dependency PR first: #17387 into `trunk`
   (⚠️ Note: That PR also has a dependency.)
2. Remove `Not Ready for Merge` flag
3. Merge this PR

## Regression Notes
1. Potential unintended areas of impact
  N/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
  N/a

3. What automated tests I added (or what prevented me from doing so)
  None for now - planned for other PRs.

PR submission checklist:
